libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.scss

```diff
@@ -69,66 +127,84 @@
   .sky-summary-action-bar-summary {
     order: -1;
     width: 100%;
-    @include defaultMixins.sky-border(light, bottom);
+    border-bottom: var(
+      --sky-override-summary-action-bar-summary-collapsible-border-bottom,
+      none
+    );
     margin: 0;
     border-left: none;
     z-index: 998;
   }
 
   .sky-summary-action-bar-summary-body {
-    padding: $sky-padding;
+    padding: var(
+      --sky-override-summary-action-bar-summary-body-collapsible-padding,
+      var(--sky-space-inset-pillarbox-2_3-top-m)
+        var(--sky-space-inset-pillarbox-2_3-right-m)
+        var(--sky-space-inset-pillarbox-2_3-bottom-m)
+        var(--sky-space-inset-pillarbox-2_3-left-m)
+    );
   }
 
   .sky-summary-action-bar-details-expand,
   .sky-summary-action-bar-details-collapse {
     display: block;
   }
-}
 
-@include defaultMixins.sky-theme-modern {
-  .sky-summary-action-bar-actions {
-    padding: $sky-theme-modern-space-lg $sky-theme-modern-space-xl;
-  }
-
-  .sky-summary-action-bar-summary {
-    border: none;
+  .sky-summary-action-bar-details-expand {
+    margin-left: auto;
+    margin-right: var(
+      --sky-override-summary-action-bar-details-expand-margin-right,
+      var(--sky-space-inline-xl)
+    );
   }
 
-  .sky-summary-action-bar-summary-body {
-    padding-left: $sky-theme-modern-space-xl;
+  .sky-summary-action-bar-details-collapse {
+    margin-left: var(
+      --sky-override-summary-action-bar-details-collapse-margin-left,
+      var(--sky-space-gap-text_action-l)
+    );
+    margin-right: var(
+      --sky-override-summary-action-bar-details-collapse-margin-right,
+      calc(
+        var(--sky-space-inline-xl) - var(
+            --sky-space-inset-pillarbox-2_3-right-m
+          )
+      )
+    );
   }
 
-  .sky-summary-action-bar-summary-collapsible {
-    .sky-summary-action-bar-actions {
-      padding: $sky-theme-modern-padding-even-lg;
-    }
-
-    .sky-summary-action-bar-summary-body {
-      padding: $sky-theme-modern-padding-even-lg;
-    }
+  .sky-summary-action-bar-actions {
+    padding: var(
+      --sky-override-summary-action-bar-summary-actions-collapsible-padding,
+      var(--sky-space-inset-pillarbox-2_3-top-m)
+        var(--sky-space-inset-pillarbox-2_3-right-m)
+        var(--sky-space-inset-pillarbox-2_3-bottom-m)
+        var(--sky-space-inset-pillarbox-2_3-left-m)
+    );
   }
+}
 
+@include defaultMixins.sky-theme-modern {
   .sky-summary-action-bar-summary-items {
-    margin-bottom: -$sky-theme-modern-margin-inline-sm;
+    // NOTE: This is to maintain backwards compatibility with v1 modern. We however, were removing extra space when key infos are not used.
```

Comment:
I found this when looking at the key-info stuff. We were removing space that wasn't actually added when consumers don't use a key info. I maintained current behavior in default and modern, but moved the space removal to the new `has` selector in v2.